packages/boot/tools/drivers.ts

```diff
@@ -224,37 +224,48 @@ export const makeGovernanceDriver = async (
     ),
   );
 
+  const findInvitation = (wallet, descriptionSubstr) => {
+    return wallet
+      .getCurrentWalletRecord()
+      .purses[0].balance.value.find(v =>
+        v.description.startsWith(descriptionSubstr),
+      );
+  };
+
   const ecMembers = smartWallets.map(w => ({
     ...w,
     acceptCharterInvitation: async (
       charterOfferId = charterMembershipId,
       instance = agoricNamesRemotes.instance.econCommitteeCharter,
     ) => {
-      await w.executeOffer({
-        id: charterOfferId,
-        invitationSpec: {
-          source: 'purse',
-          instance,
-          description: 'charter member invitation',
-        },
-        proposal: {},
-      });
+      if (findInvitation(w, 'charter member invitation')) {
+        await w.executeOffer({
+          id: charterOfferId,
+          invitationSpec: {
+            source: 'purse',
+            instance,
+            description: 'charter member invitation',
+          },
+          proposal: {},
+        });
+      }
     },
     acceptCommitteeInvitation: async (
       committeeOfferId = committeeMembershipId,
       instance = agoricNamesRemotes.instance.economicCommittee,
     ) => {
-      const description =
-        w.getCurrentWalletRecord().purses[0].balance.value[0].description;
-      await w.executeOffer({
-        id: committeeOfferId,
-        invitationSpec: {
-          source: 'purse',
-          instance,
-          description,
-        },
-        proposal: {},
-      });
+      const invitation = findInvitation(w, 'Voter');
+      if (invitation) {
```

Comment:
If there's no such invitation, this seems to fail silently. Silent failures drive me buggy. How about `assert(invitation);`?

Reply:
this check was added because this function is also triggered from the liquidationDriverKit. since im now using that along with my regular tests, it results in the invitation being accepted twice.
since we dont want that, and it would be kind of unnecessary complicated to update the liquidationDriverKit (and as a result, the tests that use it), i thought it would be easier to have this check present

Reply:
rename it to `acceptOutstandingInvitation()`? and add an `else` case to `console.log('no outstanding Voter invitation')`?

For this sort of thing, I prefer an early return:

```js
  if (!invitation) {
    console.log(...);
    return;
  }
```

It avoids indenting the code for the main execution path.